frontend/build-configs/types.ts

```diff
@@ -99,6 +99,9 @@ export type CommonBuildConfigType = {
         production: string
         local: string
     }
+    applicationUrlQueryKeyName: string | null,
```

Comment:
I think it would be better to move this to separate object
```
applicationUrlQuery  {
name: string
birthday: string
referenceNumber: string} | null
```

would also make the checks easiert and we only need to set one attribute to null for the other projects

Reply:
Yes i tried this at the first place but i didn't work because this will result to an empty class for other projects in the build config dart file

Reply:
~~How about using an array, e.g.:~~
``` ts
// koblenz
applicationQueryParams: [QUERY_PARAM_NAME, QUERY_PARAM_BIRTHDAY, QUERY_PARAM_KOBLENZ_REFERENCE_NUMBER],
// bayern
applicationQueryParams: [],
```
Edit: I think not that easy to implement because then you need to check in dart code what each param key means/which field it refers to.